src/components/Messaging/SearchRecipientsModal.tsx

```diff
@@ -125,8 +124,6 @@ const defaultStyles = createStyles('SearchRecipientsModal', (theme) => {
     rootView: {
       backgroundColor: theme.colors.elevation.level1,
       height: Platform.select({ android: '90%', default: '100%' }),
-      width: '90%',
-      marginHorizontal: '5%',
```

Comment:
Spacing is now handled by the modal. Spacing on the modal was necessary to achieve the same style while also setting a background to avoid the RTCView warning.